src/benchmarks/real-world/ILLink/SampleProject/HelloWorld.template.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
```

Comment:
I think if this gets updated to target net7.0 this test should start working and we can get this checked in.